autoPyTorch/datasets/base_dataset.py

```diff
@@ -71,10 +73,10 @@ def __getitem__(self, idx: int) -> np.ndarray:
 class BaseDataset(Dataset, metaclass=ABCMeta):
     def __init__(
         self,
-        train_tensors: BaseDatasetType,
-        dataset_name: Optional[str] = None,
-        val_tensors: Optional[BaseDatasetType] = None,
-        test_tensors: Optional[BaseDatasetType] = None,
+        train_tensors: BaseDatasetInputType,
+        dataset_name: str = "",
```

Comment:
Sorry for the trouble, what I mean is to have:
```
dataset_name: str,
```
So that the dataset is a required name? If you think it should not be a required name then `Optional[str] =None`  makes sense. For looking at the code, the main code should provide it?